### PR TITLE
Update capsule thermals

### LINKED ISSUE #104: Gemini Descent mode is inverted
According to https://www.airspacemag.com/flight-today/flying-the-gusmobile-218187/?page=3, Gemini re-entered "heads down". The CoM offset of descent mode assumes a "heads up" profile.

### PR DESCRIPTION
**Pull request by Capkirk123**
Update capsule thermal properties, and patch maxTemps after RO global patches run. Everything now works mostly correctly, except for:
Gemini needs inflated skinMaxTemp due to receiving excessive convective flux. Drag cube issue?

Orion HS does not occlude Orion CM, which consequently cannot properly survive reentry. Drag cube issue.

resolves #104

### CODE REVIEW

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg [hunk 1]
```diff
@@ -117,19 +127,21 @@ PART
...
+	//Same structure as main HS, use lunar HS settings
+	//This actually survives LEO reentry on it's own. Probably not ideal
 	MODULE
 	{
 		name:NEEDS[!DeadlyReentry] = ModuleAblator
 		name:NEEDS[DeadlyReentry] = ModuleHeatShield
 		ablativeResource = Ablator
 		outputResource = CharredAblator
-		outputMult = 0.75
+		outputMult = 0.8
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg:139]: Why the change in conversion rate?  I think the RO tool uses 0.75.  I don't know the basis for that number, question is more about consistency.

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg [hunk 2]
```diff
@@ -620,3 +632,11 @@ PART
 		RFBand = S		//Default to S-band
 	}
 }
+
+//	================================================================================
+//	Override RO global settings to get our temperatures back
+//	================================================================================
+@PART[ROC-ApolloCMBlockIII]:FOR[RealismOverhaul]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg:639]: This needs to be AFTER[RealismOverhaul] since this is a separate mod.
Functionally, ROCapsules being lexicographically after RealismOverhaul means that anything we put in "FOR:[RealismOverhaul]" will run after every FOR[RealismOverhaul] patch located in the RealismOverhaul folder, anyway.
This comment applies throughout the entire PR.

File: GameData/ROCapsules/PartConfigs/DynasoarMoroz/DynaCockpitMoroz.cfg [hunk 3]
```diff
@@ -371,3 +371,11 @@ PART
...
 	}
 }
+
+//	================================================================================
+//	Override RO global settings to get our temperatures back
+//	================================================================================
+@PART[ROC-DynaCockpitMoroz]:FOR[RealismOverhaul]
+{
+	@skinMaxTemp = 2000
+}
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/DynasoarMoroz/DynaCockpitMoroz.cfg:381]: FOR -> AFTER.  And Newline before EOF

File: GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryCM.cfg [hunk 4]
```diff
@@ -523,11 +532,19 @@ PART
...
+{
+	@skinMaxTemp = 1144
+}
+
 //	================================================================================
 //	Final Pass to Make Sure TAC does not add extra resources
 //	================================================================================
 
-@PART[ROC-MercuryCM]:BEFORE[zzzRealismOverhaul]
+@PART[ROC-MercuryCMBDB]:BEFORE[zzzRealismOverhaul]
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryCM.cfg:547]: Oof.  Reminder to the rest of us that we need to be careful about when we make new parts versus just renaming something.  Yay not breaking saves.  Boo for creating patches that no longer get applied.

**Review by DRVeyl — CHANGES_REQUESTED** [review]

**Comment by DRVeyl** [comment]
Also looks like you addressed Apollo's version of #104 here.  Want to fix for Gemini?

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg (hunk 1 above)
Comment (Capkirk123) [reply GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg:139]: The RO tool uses 0.8 (at least, that's what it ends up as in the cache).

File: GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg (hunk 2 above)
Comment (Capkirk123) [reply GameData/ROCapsules/PartConfigs/Apollo ETS/ApolloCMBlockIII+.cfg]: Patching syntax corrected.

File: GameData/ROCapsules/PartConfigs/DynasoarMoroz/DynaCockpitMoroz.cfg (hunk 3 above)
Comment (Capkirk123) [reply GameData/ROCapsules/PartConfigs/DynasoarMoroz/DynaCockpitMoroz.cfg]: Added newline

File: GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryCM.cfg (hunk 4 above)
Comment (NathanKell) [reply GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryCM.cfg:547]: This looks like a comment rather than a work item, so I'm resolving.
